web-interface/vm-consoles-modules/vm-desktop-viewer-web.adoc

```diff
@@ -0,0 +1,15 @@
+[[vm-desktop-viewer-web]]
+== Using Desktop Viewer to connect to RDP Console (Windows only)
+
+The Desktop Viewer option can be used to connect to the RDP console of a running Windows virtual machine.
+
+.Procedure
+
+. Click *Workloads* -> *Virtual Machines* from the side menu.
+. Select a virtual machine.
+. Click *Consoles*. The VNC console opens by default.
+. Select *Desktop Viewer* in the drop-down list.
+. Choose the nework interface you want to connect to from the *Network Interface* drop-down.
+.. If L2 Network Interface is chosen, you will need to install qemu-guest-agent on the VM.
```

Comment:
"If L2 Network Interface" -> "If the L2 network interface"

Reply:
fixed